milter/client/meson.build

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2022  Daijiro Fukuda <fukuda@clear-code.com>
+# Copyright (C) 2022  Sutou Kouhei <kou@clear-code.com>
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+sources = files(
+  'milter-client-context.c',
+  'milter-client-main.c',
+  'milter-client-runner.c',
+  'milter-client-single-thread-runner.c',
+  'milter-client.c',
+)
+
+headers = files(
+  'milter-client-context.h',
+  'milter-client-objects.h',
+  'milter-client-runner.h',
+  'milter-client-single-thread-runner.h',
+  'milter-client.h',
+)
+
+install_headers(headers, subdir: join_paths('milter', 'client'))
+
+enums = gnome.mkenums_simple('milter-client-enum-types',
+                             body_prefix: '#include <config.h>',
+                             identifier_prefix: 'Milter',
+                             install_dir: join_paths('milter', 'client'),
+                             install_header: true,
+                             sources: headers,
+                             symbol_prefix: 'milter')
+
+gobject = dependency('gobject-2.0')
+c_compiler = meson.get_compiler('c')
+ev = declare_dependency(compile_args: '-DEV_COMPAT3=0',
+                        dependencies: c_compiler.find_library('ev'))
+dependencies = [
+  config,
+  gobject,
```

Comment:
We need to depend on `milter_core` instead.

Reply:
Thanks for the fix!